### PR TITLE
Update style-guide glossary

### LINKED ISSUE #3: Hindi glossary
Adding a list of commonly used translations

| Term | Translation |
| :---: | :---:|
| App | एप्प |
| Apps | ऍप्स |
| Javascript | जावास्क्रिप्ट |
| Tutorial | ट्यूटोरियल |
| Text | टेक्स्ट |
| Value | वैल्यू  |
| View | व्यू |

Content that does not need to be translated:

| No translation required |
| :-----: |
| API |
| array |
| class |
| constructor |
| context |
| CDN |
| HTML |
| props |
| strings |
| UI |
| key |

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Fixes #3 
-

Add glossary and no translation required words in style guide

### CODE REVIEW

File: style-guide.md [hunk 1]
```diff
@@ -84,7 +84,50 @@ For links that have no equivalent (Stack Overflow, YouTube videos, etc.), just u
...
+| Additional | एडिशनल |
+| App | एप्प |
+| Apps | ऍप्स |
+| Deploy | डेप्लॉय |
+| Directories | डायरेक्ट्रीज |
+| Execute | एग्ज़ीक्यूट |
+| Functionality | फ़ंक्शनैलिटी |
+| Javascript | जावास्क्रिप्ट |
+| Plugin | प्लगइन |
+| Professionally | प्रोफेश्नल्ली |
```
Comment (SatyaRF) [inline style-guide.md]: प्रोफेश्नल्ली => प्रोफेशनली

**Review by SatyaRF — CHANGES_REQUESTED** [review]
Hi @arshadkazmi42 requested one change.

File: style-guide.md [hunk 2]
```diff
@@ -84,7 +84,50 @@ For links that have no equivalent (Stack Overflow, YouTube videos, etc.), just u
...
+| Projects | प्रोजेक्टस |
+| Pull | पुल्ल |
+| Query | क्वेरी |
+| Repository | रिपॉजिटरी |
```
Comment (akramkazmi71) [inline style-guide.md]: रिपॉजिटरी --> रिपॉज़िटरी

File: style-guide.md (hunk 1 above)
Comment (arshadkazmi42) [reply style-guide.md]: done with the changes

**Review by akramkazmi71 — CHANGES_REQUESTED** [review]
रिपॉजिटरी --> रिपॉज़िटरी

File: style-guide.md (hunk 2 above)
Comment (arshadkazmi42) [reply style-guide.md]: fixed. thanks for reviewing

**Review by SatyaRF — APPROVED** [review]

**Review by akramkazmi71 — APPROVED** [review]

**Pull request merged**